### DISCUSSION TITLE
Ability to copy player uuid with /seen command

### DISCUSSION THREAD

**Discussion by YanisBft** [EssentialsX/Essentials#5150] [Ideas and Feature Suggestions]
As OP players can see a player uuid with /seen command, it would be so useful to be able to copy this uuid to clipboard by clicking it
Cheers

**Comment by LaxWasHere** [comment]
<https://namemc.com/>

**Reply by YanisBft** [reply]
By this logic, there's no even need to show the uuid because such website exists

**Reply by Evidentsinger14** [reply]
Is this even possible without using components (adventure is in the works) or NMS? ideally NMS should be avoided so it doesn’t break cross version.

**Reply by JRoy** [reply]
I will add this when we migrate to adventure. Keeping this open for now
